src/ctap/test_helpers/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod customization;
+pub mod env;
```

Comment:
I think `env` is a misleading name:

- `customization` is a part of `Env`.
- It doesn't convey that we want to call the CTAP API.

Maybe `ctap` is also not useful, since the crate name is `ctap`. Some variant of `state`?

Reply:
I moved out the customization test_helper. So the original env helpers are now just under the test_helper module, and we don't need to think of naming issue anymore.

Reply:
Works for me.